cloudnet-wrapper-jvm/src/main/java/de/dytanic/cloudnet/wrapper/runtime/RuntimeApplicationClassLoader.java

```diff
@@ -1,59 +0,0 @@
-package de.dytanic.cloudnet.wrapper.runtime;
-
-import java.net.URL;
-import java.net.URLClassLoader;
-
-/**
- * This class loader is a bridge which should replace
- * the AppClassLoader and access the AppClassLoader if classes are not found.
- * This allows the actual service to access the benefits of the SystemClassLoader.
- */
-public final class RuntimeApplicationClassLoader extends URLClassLoader {
-
-    static {
-        ClassLoader.registerAsParallelCapable();
-    }
-
-    private final ClassLoader cloudNetWrapperClassLoader;
-
-    public RuntimeApplicationClassLoader(ClassLoader cloudNetWrapperClassLoader, URL url, ClassLoader parent) {
-        super(new URL[]{url}, parent);
-
-        this.cloudNetWrapperClassLoader = cloudNetWrapperClassLoader;
-    }
-
-    @Override
-    protected Class<?> loadClass(String name, boolean resolve) throws ClassNotFoundException {
-        try {
-            Class<?> loaded = this.findLoadedClass(name);
-            if (loaded != null) {
-                return loaded;
-            } else {
-                return super.loadClass(name, resolve);
-            }
-
-        } catch (Throwable ex) {
-            return cloudNetWrapperClassLoader.loadClass(name);
-        }
-    }
-
-    @Override
-    public URL getResource(String name) {
-        URL url = super.getResource(name);
-
-        if (url == null) {
-            url = cloudNetWrapperClassLoader.getResource(name);
-        }
-
-        return url;
-    }
-
-    @Override
-    public boolean equals(Object obj) {
-        return super.equals(obj) || cloudNetWrapperClassLoader.equals(obj);
-    }
-
-    public ClassLoader getCloudNetWrapperClassLoader() {
-        return this.cloudNetWrapperClassLoader;
-    }
-}
```

Comment:
Reason to remove this whole class?

Reply:
everything this class has done is replaced with bether functions. Methods like getCloudNetWrapperClassLoader won't ever be call, this class is used nowhere. URLClassLoader makes every job. If this class is unnecessary and will never be needed (to use it the system class loader has to be touched -> we don't want that) it would make sence to delete it.